assets/css/mason.css

```diff
@@ -7,20 +7,37 @@ body, html {
     background-size: cover;
     */
     font-family: 'Source Sans Pro', sans-serif;
-    background-color: 	rgb(120,120,120);
+    background-color: white;
     color: white;
     height: 100%;
     margin: 0;
     overflow: scroll;
+    /*
+    background-image: url('../../images/sam-mason-homepage.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    */
+}
+
+.about-background-img {
```

Comment:
Doesn't look like it is being used here?

Reply:
Did not mean to commit this. It is going to be used in a different pull request. It would take too much time to back this out. Can we disregard for now?

Reply:
This is not impacting any consumer facing components.